packages/js/e2e-utils/src/flows/merchant.js

```diff
@@ -409,6 +410,41 @@ const merchant = {
 			await merchant.checkDatabaseUpdateComplete();
 		}
    },
+
+	/**
+	 * Dismiss the onboarding wizard if it is open.
+	 */
+	dismissOnboardingWizard: async () => {
+		let waitForNav = false;
+		const skipButton = await page.$( '.woocommerce-profile-wizard__footer-link' );
+		if ( skipButton ) {
+			await skipButton.click();
+			waitForNav = true;
+		}
+
+		// Dismiss usage tracking pop-up window if it appears on a new site
+		const usageTrackingHeader = await page.$( '.woocommerce-usage-modal button.is-secondary' );
+		if ( usageTrackingHeader ) {
+			await usageTrackingHeader.click();
+			waitForNav = true;
+		}
+
+		if ( waitForNav ) {
+			await page.waitForNavigation( { waitUntil: 'networkidle0' } );
+		}
+	},
+
+	/**
+	 * Expand or collapse the WP admin menu.
+	 * @param {boolean} collapse Flag to collapse or expand the menu. Default collapse.
+	 */
+	collapseAdminMenu: async ( collapse = true ) => {
+		const collapseButton = await page.$( '.folded #collapse-button' );
+		if ( ( ! collapseButton ) == collapseButton ) {
```

Comment:
`( ! collapseButton ) == collapseButton` seems a bit off to me. Was is intended to be `( ! collapse ) == collapseButton`?

Reply:
@MrJnrman Thanks for this one as well. This was an autocomplete oops. The second one should have been `collapse`. The `! collapseButton` ensures the value is converted to a boolean. Fixed as well.

This is ready for another review.